booking-engine-widget/getting-started.md

```diff
@@ -68,6 +68,7 @@ https://pay.datatrans.com/upp/payment/js/secure-fields-1.0.0.js
 https://www.recaptcha.net
 https://www.google.com/recaptcha/
 https://recaptcha.google.com/recaptcha/
+https://www.gstatic.com/recaptcha/
```

Comment:
Have you not included this domain on purpose? Google docs have it listed there. So I thought I'd ask this way, through PR :) https://developers.google.com/recaptcha/docs/faq#im-using-content-security-policy-csp-on-my-website.-how-can-i-configure-it-to-work-with-recaptcha

Reply:
Ouch, definitely not on purpose, I missed it 🤦